syz-manager/cover.go

```diff
@@ -112,7 +112,49 @@ func parseFile(fn string) ([][]byte, error) {
 	return lines, nil
 }
 
+func getVmOffset(vmlinux string)(uint32, error) {
+	cmd := exec.Command("readelf", "-SW", vmlinux)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return 0, err
+	}
+	defer stdout.Close()
+	if err := cmd.Start(); err != nil {
+		return 0, err
+	}
+	defer cmd.Wait()
+
+	s := bufio.NewScanner(stdout)
+	var addr uint32 = 0
+	for s.Scan() {
+		ln := s.Text()
+		pieces := strings.Fields(ln)
+		for i := 0; i < len(pieces); i++ {
+			if pieces[i] == "PROGBITS" {
```

Comment:
In Go the common agreement is to keep indentation level is shallow as possible, so:
```
  if pieces[i] != "PROGBITS" {
    continue
  }
```